pac4j-core/src/main/java/org/pac4j/core/context/session/J2ESessionStore.java

```diff
@@ -59,7 +59,7 @@ public Object getTrackableSession(final J2EContext context) {
         if (trackableSession != null) {
             return new J2EProvidedSessionStore((HttpSession) trackableSession);
         } else {
-            return null;
+            return (SessionStore<J2EContext>) SessionStore.EMPTY;
```

Comment:
I have no opinion on this, but is it really what we would want? to throw exception while before we would be returning `null`? (I haven't finished reading, so maybe I will get an answer in the rest of the diff : )

Reply:
We should return an `Optional` here as it might be `null` when this capability is not supported.